.travis.yml

```diff
@@ -11,6 +11,7 @@ before_install:
   - gem --version
   - bundle -v
 script:
+  - 'bundle exec puppet --version'
```

Comment:
this will also be overwritten by our modulesync_config. What was your intention to add it? We set the puppet version via travis args.